### DISCUSSION TITLE
Directory-level authority management

### DISCUSSION THREAD

**Discussion by waywtdcc** [seaweedfs/seaweedfs#2323] [Q&A]
Hello everyone, currently our company is doing a directory-level permission management based on seaweedfs. I don't know much, and would like to ask if there are any good solutions now?

**Comment by chrislusf** [comment]
> directory-level permission management

what is it? Manage quota or manage user access? If latter, how to get the user ids?

Or back off a bit, what is the use case?

**Reply by waywtdcc** [reply]
> > directory-level permission management
> 
> what is it? Manage quota or manage user access? If latter, how to get the user ids?
> 
> Or back off a bit, what is the use case?

It is to manage user rights. Different users have different access rights to the same directory.

**Reply by chrislusf** [reply]
currently not supported.
